docs/userguide.md

```diff
@@ -208,21 +210,28 @@ It is possible to disable propagation of a type that is configured for propagati
 `kubefed2` command:
 
 ```bash
-kubefed2 federate disable <FederatedTypeConfig name>
+kubefed2 disable <FederatedTypeConfig Name>
 ```
 
-This command will set the `propagationEnabled` field in the `FederatedTypeConfig` to `false`,
-which will prompt the sync controller for the type to be stopped.
+This command will set the `propagationEnabled` field in the `FederatedTypeConfig` 
+associated with this target API type to `false`, which will prompt the sync 
+controller to stop sync for this target API type.
 
-If the goal is to permanently disable federation of a type, passing the `--delete-from-api`
-flag will remove the `FederatedTypeConfig` and the primitive CRDS created by `enable`:
+If the goal is to permanently disable federation of the target API type, passing the 
+`--delete-from-api` flag will remove the `FederatedTypeConfig` and the associated 
+primitive CRDS, created by `enable`:
 
 ```bash
-kubefed2 federate disable <FederatedTypeConfig name> --delete-from-api
+kubefed2 disable <FederatedTypeConfig Name> --delete-from-api
 ```
 
 **WARNING: All primitive custom resources for the type will be removed by this command.**
 
+***Note***: The NAME needed in `disable` command is the name of the FederatedTypeConfig
+		created by `kubefed2 enable` for the `target kubernetes API type`. The name format used 
```

Comment:
I don't think it should be necessary to repeat the documentation of the name form here.